Source/EasyNetQ/MessagePropertiesExtensions.cs

```diff
@@ -9,17 +9,17 @@ namespace EasyNetQ;
 /// <summary>
 ///     Represents various properties of a message
 /// </summary>
-internal static class MessagePropertiesExtensions
+public static class MessagePropertiesExtensions
 {
     public const string ConfirmationIdHeader = "EasyNetQ.Confirmation.Id";
```

Comment:
public -> internal here ?

Reply:
You are right, I made it in a separate commit